README.md

```diff
@@ -205,6 +209,55 @@ Retriable.retriable on_retry: do_this_on_each_retry do
 end
 ```
 
+### RetriableContexts
+
+There is a separate plugin to enable the contexts feature.  It's not included by default; to use it do:
+
+```ruby
+require 'retriable_contexts'
```

Comment:
that makes sense; i'll definitely do that but will wait until https://github.com/kamui/retriable/pull/32 is sorted just to make sure stuff ends up in the right place